source/developer/fx-guidelines.rst

```diff
@@ -8,7 +8,7 @@ The purpose of the Mattermost Fan eXperience Guidelines ("FX Guidelines") is to
 
 |
 
-Design Principles
+Design principles
```

Comment:
Not sure why we are changing to sentence case for these headings? I think we decided in the docs guidelines headings of level H1, H2, H3 should be title case? http://docs.mattermost.com/process/documentation-guidelines.html#use-appropriate-heading-case